doc/proposals/improved-scorecard-config.md

```diff
@@ -0,0 +1,112 @@
+# Improved Scorecard Config
+
+Implementation Owner: AlexNPavel
+Status: Draft
+
+- [Background](#Background)
+- [Goals](#Goals)
+- [Design overview](#Design-overview)
+- [User facing usage](#User-facing-usage)
+
+## Background
+
+The `scorecard` subcommand in the `operator-sdk` currently has ~15 different command line flags. This creates
+unnecessary complexity for users. Also, many of the config options only apply to the internal plugins, and
+this may become confusing to users who use external plugins, as they may think that these config options
+affect their external plugins. Using a config file that configures plugins on an individual
+basis can help clarify these differences as well as make configuration a lot cleaner
+
+## Goals
+
+Drastically reduce the amount of command line flags in the `scorecard` subcommand and define a configuration file
+format that can allow for the simple configuration of both internal and external plugins.
+
+## Design overview
+
+The configuration for the scorecard will be in a `scorecard` subsection of the config file. This will allow
+the config file to continue working properly when the SDK adds global config file support for all subcommands.
+Under the scorecard subsection there are configuration options that apply to the entire scorecard as well as
+a section that allows for the configuration of both internal and external plugins. Here are the config options
+and what they do:
+
+- `kubeconfig` string - path to kubeconfig. This option sets the kubeconfig for internal plugins and sets the `KUBECONFIG` env var for external plugins
+- `output` string - sets output format. Valid values: `human-readable` and `json`
```

Comment:
Nit: Can we make this `text` instead of `human-readable`?

Reply:
The current config uses the term `human-readable`, this is just copied from that. This would probably be a good opportunity to change it though, since we're completely changing the way we configure this. I think `text` is a better naming than `human-readable`.

/cc @joelanford @estroz @shawn-hurley

Reply:
@AlexNPavel I agree with `text`